### DISCUSSION TITLE
saved views

### DISCUSSION THREAD

**Discussion by PaoloPetra** [iTwin/itwinjs-core#3120] [Q&A] [answered]
Hello hello, I would like to implement in my personal view the possibility to see the various saved views.

If I open my imodel on https://connect-imodelweb.bentley.com/
I see below a button that gives me the opportunity to see any saved views.

![saved view](https://user-images.githubusercontent.com/40106886/151348601-d07a7104-8970-46ba-aac0-4ff00b6e9664.PNG)

on the contrary if I open it from my program this button does not exist
do you need to create a widget or change the defaultUiConfig of the <view> component?

![saved view mio](https://user-images.githubusercontent.com/40106886/151348646-d329e017-5c34-4357-9d32-0c27ad83820d.PNG)

Thanks for your help

**Comment by Carl-Hinkle** [comment, marked as answer]
Hi @PaoloPetra,

We are currently working on a public API that will allow you to access and create saved views, which you'll be able to use to build your own UI.  This should be available in beta soon.  We also plan to make the design review widget available, but that will take some time and I don't have any estimation for when that'll be done.

Let us know if you have any questions.

**Reply by PaoloPetra** [reply]
perfect I await the beta version of the API to integrate it

thank you
